lib/hand_ranker.rb

```diff
@@ -0,0 +1,40 @@
+require 'hand'
+
+class HandRanker
```

Comment:
Great refactor. It makes a lot of sense to separate out these methods into a HandRanker.

Just gonna copy over my note from the other PR which now applies to this class:  I like the idea of adding up the card values to determine the rank, but there are some magic numbers floating around this class (28 & 60). Can you define those so it's easier to see what they signify? Additionally, these numbers will have to change if the card values ever change. What if you composed the sums here by adding up the relevant values from Deck's @@values array?